docs/jotai/guides/atoms-in-atom.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Atoms in atom
+nav: 11.5
```

Comment:
I would like to put this between 11 and 12. How would we do?

Reply:
Interestingly, this seems working.
So, let me merge this and let's look for better ways.